books/kestrel/crypto/r1cs/sparse/doc.lisp

```diff
@@ -14,8 +14,10 @@
 
 (include-book "kestrel/utilities/gen-xdoc-for-file" :dir :system)
 
-;; Introduced by defaggregate
+;; No longer introduced by defaggregate.
```

Comment:
This seems incorrect or at least unclear.  r1cs-constraintp is still introduced by defaggregate.

Reply:
No, the xdoc for r1cs-constraintp is generated in doc.lisp by the defxdoc immediately below the comment.  It looks to me like :suppress-xdoc is suppressing all forms of xdoc on the defaggregate.

Reply:
Oh.  Now I get what you are saying here.  I misunderstood your comment so I made the wrong change.
OK, then let's merge the defaggregate commit and ignore the other one.